parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -27,7 +27,7 @@ installDeps() {
     local OSVERSION
     OSVERSION=$(grep DISTRIB_RELEASE /etc/*-release| cut -f 2 -d "=")
     BLOBFUSE_VERSION="1.4.5"
-    BLOBFUSE2_VERSION="2.0.3"
```

Comment:
There is `2.0.3` in https://github.com/Azure/AgentBaker/blob/ae6f7253fd6b2d813106e67ac3c0ad9b300a4283/vhdbuilder/packer/install-dependencies.sh#L56

Reply:
good catch, already updated.

Reply:
seems there is a new minimal image now, interesting.